test/connectors/CustomsDataStoreConnectorSpec.scala

```diff
@@ -41,6 +41,16 @@ class CustomsDataStoreConnectorSpec extends SpecBase {
       }
     }
 
+    "return undeliverable email address from customs data store" in new Setup {
+      val emailResponse = EmailResponse(Some("noresponse@email.com"), Some("time"), Some(UndeliverableInformation("subject-example","ex-event-id-01","ex-group-id-01")))
```

Comment:
Could you please split this over multiple lines as it is very long line? same comment is applicable for line 47

Reply:
fixed